session/session.go

```diff
@@ -0,0 +1,144 @@
+package session
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// PortalData ...
+type PortalData struct {
+	AppleID              string              `json:"apple_id"`
+	Password             string              `json:"password"`
+	ConnectionExpiryDate string              `json:"connection_expiry_date"`
+	SessionCookies       map[string][]Cookie `json:"session_cookies"`
+}
+
+// Cookie ...
+type Cookie struct {
```

Comment:
this should be unexported

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/e254c7963832e39211f47b32764c9d0ff4bd246e